diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSmokeTest.kt

```diff
@@ -179,6 +169,25 @@ class DiktatSmokeTest : FixTestBase("test/smoke/src/main/kotlin",
         )
     }
 
+    @Test
+    @Tag("DiktatRuleSetProvider")
+    fun `smoke test with kts files`() {
+        overrideRulesConfig(
+            listOf(
+                HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE  // because build.gradle.kts doesn't need extra comments, and this rule can be manually disabled if needed
```

Comment:
As I understand we should disable this rule if we want to check `kts` files. What if we want to check `kt` and `kts` files together, will it trigger?

Reply:
It is not really needed for `build.gradle.kts` because it's a standard file that doesn't need docs. But for other kotlin scripts, e.g. in `buildSrc` dir, it is still useful. I think you are right, I should add some filtering for gradle-specific files only.